helping-and-teaching.md

```diff
@@ -28,34 +24,34 @@ Teaching isn't about helping those who are already "in", it is for those who are
 If you see anything that is not supporting an equal learning environment,
 please mention it to one of instructors.
 
-
 ### Creating a positive learning environment
 
-As a helper, you have a crucial role during workshops:
+As an exercise leader, you have a crucial role during workshops:
 
 - Encourage learners to learn from each other.
-- Acknowledge that some of the material can be difficult and that they will
+- Acknowledge that some of the material can be difficult and that people in your breakoutroom will
   learn more working together.
 - Acknowledge when learners are confused and raise it to the instructors.
   Understanding why learners are confused provides useful feedback for
-  instructors. You should be our eyes and ears.
+  instructors. You are our eyes and ears.
 - As we said, you don't have to know everything, just like learners don't
-  necessarily know everything (we don't know everything, either).  It's more
+  necessarily know everything (we don't know everything, either). It's more
   important to be responsive and work together.
-- Stand up and walk around, try to make rounds by everyone.  If you are
+- In an online workshop: Turn on your camera, and encourage everyone else to do so as well. Have an introductory round in the first breakout room session, to get to know your group. Whichever strategy you choose for your breakoutroom (see [here](breakout-rooms-helping)), be present and encourage learners to ask questions.
```

Comment:
@rkdarst, "[here](breakout-rooms-helping)" as well. Does internal link grammar work in the same way as MD?